### PR TITLE
en api docs of physics

### PR DESCRIPTION
**Pull request by JayceLai**
Re: cocos-creator/3d-tasks#2506

Changes:
 *

### CODE REVIEW

File: cocos/physics/framework/assets/physic-material.ts [hunk 1]
```diff
@@ -9,23 +9,28 @@ import { Asset } from '../../../core/assets/asset';
...
-    private static _idCounter: number = 0;
-
-    @property
-    private _friction = 0.1;
-
-    @property
-    private _restitution = 0.1;
+    /**
+     * @en
+     * Gets all physical material instances.
```
Comment (slackmoehrle) [inline cocos/physics/framework/assets/physic-material.ts]: physical or physics?

File: cocos/physics/framework/components/collider/box-collider-component.ts [hunk 2]
```diff
@@ -31,7 +33,7 @@ export class BoxColliderComponent extends ColliderComponent {
 
     /**
      * @en
-     * Get or set the size of the box, in local space.
+     * Gets or set the size of the box, in local space.
```
Comment (slackmoehrle) [inline cocos/physics/framework/components/collider/box-collider-component.ts]: It should be either:
Gets or sets
__or__
Get or set

File: cocos/physics/framework/components/collider/capsule-collider-component.ts [hunk 3]
```diff
@@ -30,7 +32,7 @@ export class CapsuleColliderComponent extends ColliderComponent {
 
     /**
      * @en
-     * Get or set the radius of the sphere on the capsule body, in local space.
+     * Gets or set the radius of the sphere on the capsule body, in local space.
```
Comment (slackmoehrle) [inline cocos/physics/framework/components/collider/capsule-collider-component.ts]: It should be either:
Gets or sets
__or__
Get or set

File: cocos/physics/framework/components/collider/capsule-collider-component.ts [hunk 4]
```diff
@@ -50,7 +52,7 @@ export class CapsuleColliderComponent extends ColliderComponent {
 
     /**
      * @en
-     * Get or set the capsule body is at the corresponding axial height, in local space.
+     * Gets or set the capsule body is at the corresponding axial height, in local space.
```
Comment (slackmoehrle) [inline cocos/physics/framework/components/collider/capsule-collider-component.ts]: It should be either:
Gets or sets
__or__
Get or set

File: cocos/physics/framework/components/collider/capsule-collider-component.ts [hunk 5]
```diff
@@ -68,6 +70,12 @@ export class CapsuleColliderComponent extends ColliderComponent {
         }
     }
 
+    /**
+     * @en
+     * Gets or set the capsule direction, in local space.
```
Comment (slackmoehrle) [inline cocos/physics/framework/components/collider/capsule-collider-component.ts]: It should be either:
Gets or sets
__or__
Get or set

File: cocos/physics/framework/components/collider/collider-component.ts [hunk 6]
```diff
@@ -87,7 +103,7 @@ export class ColliderComponent extends Component implements IEventTarget {
 
     /**
      * @en
-     * get or set the collider is trigger, this will be always trigger if using builtin.
+     * Gets or set the collider is trigger, this will be always trigger if using builtin.
```
Comment (slackmoehrle) [inline cocos/physics/framework/components/collider/collider-component.ts]: It should be either:
Gets or sets
__or__
Get or set

File: cocos/physics/framework/components/collider/collider-component.ts [hunk 7]
```diff
@@ -108,7 +124,7 @@ export class ColliderComponent extends Component implements IEventTarget {
 
     /**
      * @en
-     * get or set the center of the collider, in local space.
+     * Gets or set the center of the collider, in local space.
```
Comment (slackmoehrle) [inline cocos/physics/framework/components/collider/collider-component.ts]: It should be either:
Gets or sets
__or__
Get or set

File: cocos/physics/framework/components/rigid-body-component.ts [hunk 8]
```diff
@@ -313,10 +345,12 @@ export class RigidBodyComponent extends Component {
     /// PUBLIC METHOD ///
 
     /**
+     * @en
+     * Apply force to a world point. This could for example be a point on the Body surface.
```
Comment (slackmoehrle) [inline cocos/physics/framework/components/rigid-body-component.ts]: could for example --> could, for example,

File: cocos/physics/framework/components/rigid-body-component.ts [hunk 9]
```diff
@@ -325,8 +359,10 @@ export class RigidBodyComponent extends Component {
     }
 
     /**
+     * @en
+     * Apply force to a local point. This could for example be a point on the Body surface.
```
Comment (slackmoehrle) [inline cocos/physics/framework/components/rigid-body-component.ts]: could for example --> could, for example,

File: cocos/physics/framework/physics-interface.ts [hunk 10]
```diff
@@ -6,102 +6,136 @@ import { Vec3 } from '../../core/math';
...
+     * @en
+     * The collider component instance of the event owner.
      * @zh
      * 触发事件中的自己的碰撞器
      */
     readonly selfCollider: ColliderComponent;
 
     /**
+     * @en
+     * The aother collider component instance in the event.
```
Comment (slackmoehrle) [inline cocos/physics/framework/physics-interface.ts]: The aother collider component instance in the event.
-->
Trigger another collider in event.

File: cocos/physics/framework/physics-interface.ts [hunk 11]
```diff
@@ -6,102 +6,136 @@ import { Vec3 } from '../../core/math';
...
+     * @en
+     * The collider component instance of the event owner.
      * @zh
      * 碰撞中的自己的碰撞器。
      */
     readonly selfCollider: ColliderComponent;
 
     /**
+     * @en
+     * The aother collider component instance in the event.
```
Comment (slackmoehrle) [inline cocos/physics/framework/physics-interface.ts]: The aother collider component instance in the event.
-->
Another collider in collision.

File: cocos/physics/framework/physics-ray-result.ts [hunk 12]
```diff
@@ -7,8 +7,10 @@ import { ColliderComponent } from '../../../exports/physics-framework';
 import { IVec3Like } from '../../core/math/type-define';
 
 /**
+ * @en
+ * Used to store physical ray test results.
```
Comment (slackmoehrle) [inline cocos/physics/framework/physics-ray-result.ts]: physical or physics?

**Review by slackmoehrle — CHANGES_REQUESTED** [review]
Just a few changes. Nice work.

只是一些更改。 辛苦了

**Comment by JayceLai** [comment]
> Just a few changes. Nice work.
> 
> 只是一些更改。 辛苦了

thank you.

File: cocos/physics/framework/components/collider/collider-component.ts (hunk 6 above)
Comment (slackmoehrle) [reply cocos/physics/framework/components/collider/collider-component.ts]: no, sorry. singular for both or plural for both.
Gets or sets
__or__
Get or set

两个词都必须是单数或两个词都必须是复数.

File: cocos/physics/framework/components/collider/collider-component.ts (hunk 7 above)
Comment (slackmoehrle) [reply cocos/physics/framework/components/collider/collider-component.ts]: no, sorry. singular for both or plural for both.
Gets or sets
__or__
Get or set

两个词都必须是单数或两个词都必须是复数.

**Review by slackmoehrle — CHANGES_REQUESTED** [review]
just 2 changes left. I think we may have had a communication snafu.

**Comment by JayceLai** [comment]
> just 2 changes left. I think we may have had a communication snafu.

Sorry, I missed these two changes.

File: cocos/physics/framework/components/collider/collider-component.ts (hunk 7 above)
Comment (JayceLai) [reply cocos/physics/framework/components/collider/collider-component.ts]: this is outdated, please review newest commits.

File: cocos/physics/framework/components/collider/collider-component.ts (hunk 6 above)
Comment (JayceLai) [reply cocos/physics/framework/components/collider/collider-component.ts]: this is outdated, please review newest commits.

**Review by slackmoehrle — APPROVED** [review]
Nice work. Thank you for your effort.

辛苦了 感谢你的付出。

**Comment by slackmoehrle** [comment]
@pandamicro this is ready for merge.

**Comment by pandamicro** [comment]
Thank you both for the great translation and review

**Pull request merged**